File: app/src/main/java/com/eam/investire/helper/DateCustom.java
package com.eam.investire.helper;

import java.text.SimpleDateFormat;
import java.util.Locale;

public class DateCustom {
    public static String dataAtual() {
        long data = System.currentTimeMillis();
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd/MM/yyyy", Locale.getDefault());
        return simpleDateFormat.format(data);
    }

    public static String mesAnoDataEscolhida(String data) {
        String[] retornoData = data.split("/");
        String mes = retornoData[1];
        String ano = retornoData[2];
        return mes + ano;
    }
}

File: app/src/main/java/com/eam/investire/helper/DateCustom.kt
package com.eam.investire.helper

import java.text.SimpleDateFormat
import java.util.*

object DateCustom {
    fun dataAtual(): String {
        val data = System.currentTimeMillis()
        val simpleDateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
        return simpleDateFormat.format(data)
    }

    @JvmStatic
    fun mesAnoDataEscolhida(data: String): String {
        val retornoData = data.split("/").toTypedArray()
        val mes = retornoData[1]
        val ano = retornoData[2]
        return mes + ano
    }
}